src/theme/theme.stories.mdx

```diff
@@ -1,57 +1,153 @@
+
+<Meta title="Theme" />
 
 # Theme
 
-## Colour palette
+## Color palette
 
 <Canvas>
-  <Story name="colours">
+  <Story name="colors">
     {() => {
+      const motoScout = motoScout24Theme.colors.brand;
+      const autoScout = autoScout24Theme.colors.brand;
+      const sharedColor = autoScout24Theme.colors;
       return (
-        <Flex direction="row" align="center" justify="left" wrap="wrap">
-          <Flex direction="column" align="center" justify="left" mr={5}>
-            {[
-              '50',
-              '100',
-              '200',
-              '300',
-              '400',
-              '500',
-              '600',
-              '700',
-              '800',
-              '900',
-            ].map((shade) => {
-              return (
-                <Flex
-                  direction="row"
-                  align="center"
-                  justify="space-between"
-                  key={`brand-${shade}`}
-                >
-                  <Box
-                    bg={`brand.${shade}`}
-                    h={50}
-                    w={50}
-                    mr={3}
-                    mb={3}
-                    borderRadius={50}
-                    borderWidth={1}
-                    borderStyle="solid"
-                    borderColor="gray.500"
-                  />
-                  <Box w={100}>{`brand-${shade}`}</Box>
-                </Flex>
-              );
-            })}
-          </Flex>
-        </Flex>
+        <ColorPalette>
+          <ColorItem
+            title="Black"
+            colors={{
+              black: sharedColor.black,
+            }}
+          />
+          <ColorItem
+            title="White"
+            colors={{
+              white: sharedColor.white,
+            }}
+          />
+          <ColorItem
+            title="PRIMARY"
+            subtitle="AutoScout 24"
+            colors={{
+              'brand-50': autoScout[50],
+              'brand-100': autoScout[100],
+              'brand-200': autoScout[200],
+              'brand-300': autoScout[300],
+              'brand-400': autoScout[400],
+              'brand-500': autoScout[500],
+              'brand-600': autoScout[600],
+              'brand-700': autoScout[700],
+              'brand-800': autoScout[800],
+              'brand-900': autoScout[900],
+            }}
+          />
+          <ColorItem
+            title="PRIMARY"
+            subtitle="MotoScout 24"
+            colors={{
+              'brand-50': motoScout[50],
+              'brand-100': motoScout[100],
+              'brand-200': motoScout[200],
+              'brand-300': motoScout[300],
+              'brand-400': motoScout[400],
+              'brand-500': motoScout[500],
+              'brand-600': motoScout[600],
+              'brand-700': motoScout[700],
+              'brand-800': motoScout[800],
+              'brand-900': motoScout[900],
+            }}
+          />
```

Comment:
Would it be possible to only show one brand colour depending on which theme is currently selected via the theming addon? IT should be possible to get it via `useTheme` hook from chakra

Reply:
I would apply this idea/suggestion for the components itself. I think, it'd make more sense the selected theme there.

I'll suggest for this palette view to have all the colors without selecting the theme. If we want to make it selected, we can always address it latter on.

Reply:
The thing is that only one brand colour is available per theme. My concern is that if both are visible in the theme showcase it can create a false assumption of both being always available. What do you think?

Reply:
Mmmm I see... oki doki, I will adapt the theme there. Thanks! 🙌🏼